features/cellular/TESTS/api/cellular_network/main.cpp

```diff
@@ -155,6 +155,11 @@ static void test_other()
 {
     TEST_ASSERT(nw->get_3gpp_error() == 0);
 
+#ifdef TARGET_UBLOX_C030
+    nsapi_error_t err = nw->detach();
```

Comment:
test should not contain `#ifdef target`, either works or not - handling this should be in the target code or driver.

Reply:
Removed target specific code.

Reply:
will it work (test pass) ?

Reply:
Yes, handled in driver, return `UNSUPPORTED` if try to change `RAT` while in attach state.